acorn/src/parseutil.js

```diff
@@ -152,3 +152,15 @@ pp.isSimpleAssignTarget = function(expr) {
     return this.isSimpleAssignTarget(expr.expression)
   return expr.type === "Identifier" || expr.type === "MemberExpression"
 }
+
+pp.expectPlugin = function(pluginName) {
```

Comment:
Looks like this could just be `if (!this.hasPlugin(pluginName)) this.raise(...)`, since the return value isn't useful.